docs/tutorials/authoring.md

```diff
@@ -0,0 +1,57 @@
+# Authoring Tutorial
+
+This tutorial provides an overview of the authoring process using `trestlebot`.  We will use the component definition created in the [GitHub tutorial](https://redhatproductsecurity.github.io/trestle-bot/tutorials/github/) as our starting point.  This tutorial will demonstrate the workflow for updating Markdown content and syncing those changes to OSCAL.
+
+## 1. Prerequisites
+
+- Complete the [GitHub tutorial](https://redhatproductsecurity.github.io/trestle-bot/tutorials/github/)
+
+
+## 2. Edit in Markdown
+
+We will begin where we left off at the end of the [GitHub tutorial](https://redhatproductsecurity.github.io/trestle-bot/tutorials/github/).  Our repository has a newly created component definition named `my-first-compdef` with corresponding content in the `markdown/` and `component-definitions/` directories.  We will now deomonstrate how to author changes in Markdown and produce updated OSCAL content.
+
+1. Navigate to the `markdown/component-definitions/my-first-compdef/test-component/nist_rev5_800_53/ac` directory and select the `ac-1.md` file.
+2. Click the `Edit this file` (pencil) icon.
+3. Scroll down to the section titled `## What is the solution and how is it implemented?` and add a new line of text with a brief comment.  For example:
+
+```
+## What is the solution and how is it implemented?
+
+Here is where details should be added by the author.
+```
+
+4. Click the `Commit changes..` button
+5. Select the `Create a new branch for this commit and start a pull request` radio buton
+6. Click `Propose changes`
+
+
+The `Open a pull request` page now opens.  Enter any additional details about your changes into the description box.
+
+7. Click `Create pull request`
+8. For demo purposes, we will go ahead and merge the pull request ourselves.  Click `Merge pull request` and then `Confirm merge`.
```

Comment:
Perhaps here is would be beneficial to go into what would happen in a typical scenario as well